runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/foreign_investments.rs

```diff
@@ -1450,10 +1477,96 @@ mod mismatching_currencies {
 
 	use super::*;
 	use crate::{
-		liquidity_pools::pallet::development::{setup::CHARLIE, tests::register_usdt},
+		liquidity_pools::pallet::development::{
+			setup::CHARLIE,
+			tests::{
+				liquidity_pools::foreign_investments::setup::enable_usdt_trading, register_usdt,
+			},
+		},
 		utils::{GLMR_CURRENCY_ID, USDT_CURRENCY_ID},
 	};
 
+	#[test]
+	fn collect_for() {
+		TestNet::reset();
+		Development::execute_with(|| {
+			setup_pre_requirements();
+			let pool_id = DEFAULT_POOL_ID;
+			let investor: AccountId =
+				AccountConverter::<DevelopmentRuntime, LocationToAccountId>::convert((
+					DOMAIN_MOONBEAM,
+					BOB,
+				));
+			let pool_currency: CurrencyId = AUSD_CURRENCY_ID;
+			let foreign_currency: CurrencyId = USDT_CURRENCY_ID;
+			let pool_currency_decimals = currency_decimals::AUSD;
+			let invest_amount_pool_denominated: u128 = 6_000_000_000_000_000;
```

Comment:
can we use the `dollar(<decimals>)` function here? So this would become `6 * dollar(18)` (where 18 is the AUSD decimals. Even better if we have a constant for that). Just to facilitate reading these amounts and to avoid a situation where for example we may be moving amounts that are too high or too low and not quite testing things properly.